TJDropbox/TJDropbox.m

```diff
@@ -385,6 +389,16 @@ + (void)getSharedLinkForFileAtPath:(NSString *const)path accessToken:(NSString *
     }];
 }
 
+#pragma mark - Users
+
++ (void)getSpaceUsageForUserWithAccessToken:(NSString *const)accessToken completion:(void (^const)(NSDictionary *_Nullable parsedResponse, NSError *_Nullable error))completion
+{
+    NSURLRequest *const request = [self apiRequestWithPath:@"/2/users/get_space_usage" accessToken:accessToken parameters:nil];
+    [self performAPIRequest:request withCompletion:^(NSDictionary * _Nullable parsedResponse, NSError * _Nullable error) {
+        completion(parsedResponse, error);
+    }];
```

Comment:
Looks great! Tried it out and works great too. One bit of feedback, instead of creating a second block and calling `completion` from it, can you just pass `completion` directly to `+performAPIRequest:withCompletion:` like so?

```
[self performAPIRequest:request withCompletion:completion];
```